packages/financial-templates-lib/src/logger/SlackTransport.ts

```diff
@@ -186,19 +192,32 @@ class SlackHook extends Transport {
       // If the log contains a notification path then use a custom slack webhook service. This lets the transport route to
       // different slack channels depending on the context of the log.
       const webhookUrl = this.escalationPathWebhookUrls[info.notificationPath] ?? this.defaultWebHookUrl;
-
-      const payload: { blocks?: Block[]; text?: string; mrkdwn?: boolean } = { mrkdwn: this.mrkdwn };
       const layout = this.formatter(info);
-      payload.blocks = layout.blocks || undefined;
-      // If the overall payload is less than 3000 chars then we can send it all in one go to the slack API.
-      if (JSON.stringify(payload).length < SLACK_MAX_CHAR_LIMIT) {
-        await this.axiosInstance.post(webhookUrl, payload);
-      } else {
-        // Iterate over each message to send and generate a axios call for each message.
-        for (const processedBlock of processMessageBlocks(payload.blocks)) {
-          payload.blocks = processedBlock;
-          await this.axiosInstance.post(webhookUrl, payload);
-        }
+
+      // Deliver in chunks if the the combined payload is more than SLACK_MAX_CHAR_LIMIT chars.
+      const payloads: (SlackFormatterResponse & { text?: string; mrkdwn?: boolean })[] =
+        JSON.stringify(layout.blocks).length < SLACK_MAX_CHAR_LIMIT
```

Comment:
this chunk is just a refactor of the above with no logical changes right? If so, then do you think you need to handle the case where `layout?.blocks` is undefined like it does previously?